### PR TITLE
Morph Holesky Testnet Update

### PR DESCRIPTION
**Pull request by RedHorse823**
Update Morph Holesky Testnet Informations:
1. Network Setup
2. New bridge system
3. New System Design

### CODE REVIEW

**Review by EnderCrypto — APPROVED** [review]

File: docs/quick-start/2-wallet-setup.md [hunk 1]
```diff
@@ -20,37 +20,30 @@ TBD
...
-You can use any of the following 3 rpc for connections:
-
-~~~
-https://rpc-testnet.morphl2.io
-
-https://rpc1-testnet.morphl2.io
-
-https://rpc2-testnet.morphl2.io
-~~~
+Morph Sepolia testnet is going to be sunset soon, we strongly suggest that you migrate to Morph Holesky testnet
```
Comment (EnderCrypto) [inline docs/quick-start/2-wallet-setup.md:36]: Morph Sepolia testnet is going to be sunset soon. We strongly suggest that you migrate to Morph Holesky testnet.
-- change "," to "."

File: docs/quick-start/2-wallet-setup.md [hunk 2]
```diff
@@ -20,37 +20,30 @@ TBD
...
 
 :::
 
 
 | Name                      | RPC Url(s)                            | Chain ID | Block explorer             | Symbol |
 | -------- | -------------------------- | ------------- | ---------- | ------------------------------------ |
-| Morph Testnet             | https://rpc-testnet.morphl2.io        | 2710    | https://explorer-testnet.morphl2.io      | ETH      |
+| Morph Sepolia Testnet             | https://rpc-testnet.morphl2.io        | 2710    | https://explorer-testnet.morphl2.io      | ETH      |
+| Morph Holesky Testnet             | https://rpc-holesky.morphl2.io        | 2810    | https://explorer-holesky.morphl2.io      | ETH      |
```
Comment (EnderCrypto) [inline docs/quick-start/2-wallet-setup.md:44]: Move Morph Holesky to the first

File: docs/quick-start/2-wallet-setup.md [hunk 3]
```diff
@@ -20,37 +20,30 @@ TBD
...
 | Ethereum Sepolia            | https://eth-sepolia-public.unifra.io       | 11155111    | https://sepolia.etherscan.io      | ETH      |
+| Ethereum Holesky            | https://ethereum-holesky-rpc.publicnode.com/o       | 17000    | https://holesky.etherscan.io      | ETH      |
```
Comment (EnderCrypto) [inline docs/quick-start/2-wallet-setup.md:46]: Move Ethereum Holesky to the second place

**Review by Fokoli — COMMENTED** [review]
great

**Pull request merged**